From 25d842ed0d8cc82ba025f93744bf06bc25ccf3ce Mon Sep 17 00:00:00 2001
From: juleq <julian.quindt@gmail.com>
Date: Mon, 27 Jan 2014 21:15:04 +0100
Subject: [PATCH 163/174] config: CONFIG_RT2800USB_RT55XX for more RaLink ID

I added CONFIG_RT2800USB_RT55XX=y to have my el cheapo dual band wlan adapter (RaLink chip, MediaTek brand, 148f:5572) supported by RT2800. I tested with 3.10.27+ and it works. 5GHz for 5-10 Euro.
---
 arch/arm/configs/bcmrpi_defconfig | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -488,6 +488,7 @@ CONFIG_RT2500USB=m
 CONFIG_RT73USB=m
 CONFIG_RT2800USB=m
 CONFIG_RT2800USB_RT53XX=y
+CONFIG_RT2800USB_RT55XX=y
 CONFIG_RT2800USB_UNKNOWN=y
 CONFIG_ZD1211RW=m
 CONFIG_MWIFIEX=m
